src/main/java/com/exasol/adapter/document/mapping/IterationIndexColumnMapping.java

```diff
@@ -1,16 +1,21 @@
 package com.exasol.adapter.document.mapping;
 
-import java.util.Objects;
-
 import com.exasol.adapter.document.documentpath.DocumentPathExpression;
 import com.exasol.adapter.metadata.DataType;
 
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
 /**
  * This class defines a column that maps the array index of a nested list. Such columns are useful for nested tables
  * that do not have an key.
  */
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@Data
+@SuperBuilder(toBuilder = true)
 public final class IterationIndexColumnMapping extends AbstractColumnMapping {
-    private static final long serialVersionUID = -2526873217496416853L;//
+    private static final long serialVersionUID = -5720702055496015560L;//
```

Comment:
```suggestion
    private static final long serialVersionUID = -5720702055496015560L;
```

Reply:
You have it some of them but not all, so please remove them to keep consistent if not necessary.